src/arns.lua

```diff
@@ -140,14 +189,36 @@ function arns.extendLease(from, name, years, currentTimestamp, msgId, fundFrom)
 	}
 end
 
+--- Calculates the extension fee for a given base fee, years, and demand factor
+--- @param baseFee number The base fee for the name
+--- @param years number The number of years
+--- @param demandFactor number The demand factor
+--- @return number The extension fee
```

Comment:
same as above. maybe cleaner like `<type> <name>` since name and description are nearly the same 
```suggestion
--- @return number extensionFee
```

Reply:
addressed a good chunk in [cebb3cb](https://github.com/ar-io/ar-io-network-process/pull/175/commits/cebb3cba577aa5a936939eb7bc99e01da74ebb9f)